storymap.js

```diff
@@ -113,6 +113,10 @@
                     map.setView(initPoint, initZoom, true);
                 } else if (markers[key]) {
                     var marker = markers[key];
+                    var layer = marker.layer;
+                    if(typeof layer!='undefined'){
```

Comment:
add some more spaces here:  if (typeof layer != 'undefined') {
and, use double unequality: typeof layer !== 'undefined'